front/src/applications/operationalStudies/hooks.ts

```diff
@@ -26,12 +27,11 @@ import type { ManageTrainSchedulePathProperties } from './types';
  * Hook to relaunch the pathfinding when editing a train
  */
 export const useSetupItineraryForTrainUpdate = (
-  setPathProperties: (pathProperties: ManageTrainSchedulePathProperties) => void
+  setPathProperties: (pathProperties: ManageTrainSchedulePathProperties) => void,
+  trainIdToEdit: number
 ) => {
-  const { getInfraID, getTrainScheduleIDsToModify, getUsingElectricalProfiles } =
```

Comment:
Fyi : `trainScheduleIDsToModify` was originally an array of ids because the endpoint to update trainschedules takes an array of trainschedule ids. Clients asked to be able to edit several trains at ones but the feature has never been implemented. But as it's not in the roadmap, I think it make sense that the new state takes only one id instead of an array. It can be changed later if needed.

Reply:
Actually getTrainScheduleIDsToModify is still used in TSV1, so we will remove it totally when dropping v1.

Reply:
Cool, let's keep this for later then!